README.md

```diff
@@ -51,6 +51,12 @@ need to insert data into the database. There is another make target, `make apisq
 which open the `psql` and connect to the database. Thus, you can insert data
 using some `INSERT INTO ...` statements and test the API. ;)
 
+
+### Using suggestion endpoint
+
+You need to create a token at [Mailjet](www.mailjet.com) to run
+application and send email (put on `config/current.env`).
```

Comment:
Is it possible to run without mailjet token? I mean, if I just want to test other endpoints, can I still do it?

Reply:
It's a good question. :metal: 
But Yes! I tested now and other endpoints are working :smile: